html/index.php

```diff
@@ -66,7 +68,7 @@
 <span id="last_ajax"></span>
 <script>
     var clockState = 'small';
-    clockState = $( "#datetime" ).click(function() {
```

Comment:
if we indeed don't need this `clockState = `, then can you remove the `return clockState;` too please?